index.d.ts

```diff
@@ -1628,6 +1648,7 @@ declare namespace Eris {
     createGuild(name: string, options?: CreateGuildOptions): Promise<Guild>;
     createGuildEmoji(guildID: string, options: EmojiOptions, reason?: string): Promise<Emoji>;
     createGuildFromTemplate(code: string, name: string, icon?: string): Promise<Guild>;
+    createGuildSticker(guildID: string, options: { description?: string, file: Omit<MessageFile, "name">, name: string, tags: string }, reason?: string): Promise<Sticker>;
```

Comment:
`file` isn't a nested object

Reply:
It is rn, but I could probably change that and make a completely separate interface for it if you want. I made it nested to be consistent with how the MessageFile works, but it doesn't actually *need* to be like that since it doesn't require a name or anything.

Reply:
Sorry, bad wording, meant to say shouldn't be

Reply:
It follows the same structure as a MessageFile because the request handler is built to accept a file object which transforms the Content-Type to multipart/form-data, which is what Discord wants for that endpoint